graphql/tests/hostEvents/queries/host-events.graphql

```diff
@@ -0,0 +1,30 @@
+{
+  hostEvents(hostId: "i-092593689871a50dc", hostTag: "") {
```

Comment:
if hostTag can be an empty string then should it be an optional parameter instead of a required one?